handlers/africastalking/handler_test.go

```diff
@@ -141,33 +140,32 @@ var outgoingTestCases = []OutgoingTestCase{
 		ExpectedRequests: []ExpectedRequest{
 			{Form: url.Values{"message": {"My pic!\nhttps://foo.bar/image.jpg"}, "username": {"Username"}, "to": {"+250788383383"}, "from": {"2020"}}},
 		},
-		ExpectedMsgStatus:  "W",
 		ExpectedExternalID: "1002",
 		SendPrep:           setSendURL,
 	},
 	{
-		Label:              "No External Id",
-		MsgText:            "No External ID",
+		Label:              "Explicit failed status",
+		MsgText:            "Hi",
 		MsgURN:             "tel:+250788383383",
 		MockResponseBody:   `{ "SMSMessageData": {"Recipients": [{"status": "Failed" }] } }`,
 		MockResponseStatus: 200,
 		ExpectedRequests: []ExpectedRequest{
-			{Form: url.Values{"message": {`No External ID`}, "username": {"Username"}, "to": {"+250788383383"}, "from": {"2020"}}},
+			{Form: url.Values{"message": {`Hi`}, "username": {"Username"}, "to": {"+250788383383"}, "from": {"2020"}}},
 		},
-		ExpectedMsgStatus: "E",
-		SendPrep:          setSendURL,
+		ExpectedError: courier.ErrSendResponseUnexpected,
+		SendPrep:      setSendURL,
 	},
 	{
-		Label:              "Error Sending",
+		Label:              "Missing status value",
 		MsgText:            "Error Message",
 		MsgURN:             "tel:+250788383383",
 		MockResponseBody:   `{ "error": "failed" }`,
 		MockResponseStatus: 401,
 		ExpectedRequests: []ExpectedRequest{
 			{Form: url.Values{"message": {`Error Message`}, "username": {"Username"}, "to": {"+250788383383"}, "from": {"2020"}}},
 		},
-		ExpectedMsgStatus: "E",
```

Comment:
Can we keep the `ExpectedMsgStatus` for these tests as that was brought back?
@rowanseymour

Reply:
that field will only be used for tests that use the legacy Send method type - in the new Send methods handlers don't determine the msg status - they return an error or not... and the sender chooses the right status.